README.md

```diff
@@ -142,16 +188,18 @@ you can't get info on particular field by its ID using only API. You have also s
 But with `\Badoo\Jira\REST\Client` you can do this
 
 ```php
-$Client = new \Badoo\Jira\REST\Client('https://jira.example.com/');
-$Client->setAuth('user', 'password/token');
-
+/**
+ * @var $Client \Badoo\Jira\REST\Client
+ */
 $FieldInfo = $Client->field()->get('summary');
 print_r($FieldInfo);
 ```
 
 When you can't find something in structured client, you still can access Raw client inside it to do everything you need:
 ```php
-$Client = \Badoo\Jira\REST\Client::instance();
+/**
+ * @var $Client \Badoo\Jira\REST\Client
```

Comment:
Although, popular IDEs are smart enough to digest different orderings in doc blocks, I'd use the [standard order defined by phpDocumentor](https://docs.phpdoc.org/latest/references/phpdoc/tags/var.html) as `@var Type $Variable`

Reply:
fixed